src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5074,6 +5074,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefReturnParameter2" xml:space="preserve">
     <value>Cannot return by reference a member of parameter '{0}' because it is not a ref or out parameter</value>
   </data>
+  <data name="ERR_RefReturnScopedParameter" xml:space="preserve">
+    <value>Cannot return a parameter by reference '{0}' because it is scoped to the current method</value>
+  </data>
+  <data name="ERR_RefReturnScopedParameter2" xml:space="preserve">
```

Comment:
>ERR_RefReturnScopedParameter2 [](http://example.com/codeflow?start=14&length=29)

nit: maybe `ERR_RefReturnScopedParameterMember` (instead of "2")

Reply:
I'd like to match the convention of `ERR_RefReturnParameter` and `ERR_RefReturnParameter2` for now, and perhaps change them all together later on, including improving the wording of the diagnostic message.

Reply:
Main reason to wait for later on is that I have more work coming in based on this. Once the churn starts to slow down a bit, we can take care of all this in a find/replace.